notificationworkerlambda/src/main/scala/com/gu/notifications/worker/AndroidSender.scala

```diff
@@ -22,4 +31,24 @@ class AndroidSender extends SenderRequestHandler[FcmClient] {
   override val deliveryService: IO[Fcm[IO]] =
     FcmClient(config.fcmConfig).fold(e => IO.raiseError(e), c => IO.delay(new Fcm(c)))
   override val maxConcurrency = 100
+
+  //override the deliverChunkedTokens method to validate the success of sending batch notifications to the FCM client. This implementation could be refactored in the future to make it more streamlined with APNs
+  override def deliverChunkedTokens(chunkedTokenStream: Stream[IO, (ChunkedTokens, Long, Instant)]): Stream[IO, Unit] = {
```

Comment:
Thinking about how to test/experiment in production, we could think about implementing a switch here too:
- define a list of allowed topics
- only allowed topics are sent by the batch method
- increase allowed topic list as confidence increases